pkgs/by-name/ma/mautrix-slack/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildGoModule, olm }:
+
+buildGoModule {
+  pname = "mautrix-slack";
+  version = "unstable-2024-03-19";
```

There is a new convention

```suggestion
  version = "0-unstable-2024-03-19";
```

and please reword the commit to use this version